src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/NPathComplexityCheck.java

```diff
@@ -119,14 +119,51 @@
  * &lt;module name="NPathComplexity"/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public void foo() { // NPath Complexity = 3 (OK)
```

Comment:
Please change to ` // OK, NPath complexity is less than default threshold`

Reply:
done